### ISSUE TITLE
primevue/useconfirm not found

### ISSUE THREAD

**Issue by hognevevle** [primefaces/primevue#839]
```
import { useConfirm } from "primevue/useconfirm";
```

This currently does not work per [the instructions](https://primefaces.org/primevue/showcase/#/confirmpopup). I'm on 3.1.1. 

```
 ERROR  Failed to compile with 1 error                                                                                                               5:10:17 PM

This dependency was not found:

* primevue/useconfirm in ./node_modules/cache-loader/dist/cjs.js??ref--12-0!./node_modules/babel-loader/lib!./node_modules/cache-loader/dist/cjs.js??ref--0-0!./node_modules/vue-loader-v16/dist??ref--0-1!./src/components/UserMenu.vue?vue&type=script&lang=js

To install it, you can run: npm install --save primevue/useconfirm
```

**Comment by hognevevle** [comment]
Looks like `"primevue/useconfirm"` should instead be `"primevue/useConfirm"`.

**Comment by cagataycivici** [comment]
Resolved in 3.1.2
